README.md

```diff
@@ -96,7 +96,7 @@ Name | Type | Required | Default  | Description
 --- | --- | --- | --- | ---
 **space_id**             | `string`  | Yes | `undefined` | The id of the contentful space
 **management_api_key**   | `string`  | Yes | `undefined` | The management-api key for contentful
-actions                  | `string[]`| No  | `createEnvironment, createCDAToken, applyMigration, updateAlias, cleanUpEnvironments` | The actions to run on this workflow. Comma separated string
+actions                  | `string[]`| No  | `createEnvironment, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments` | The actions to run on this workflow. Comma separated string. One of `createEnvironment, backupEnvironemnt, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments`
```

Comment:
Minor issue but `backupEnvironment` should probably also be added to the description, so:
`backupEnvironment, createEnvironment, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments`
and `backupEnvironemnt` is misspelled 😅

Reply:
The first part is the default, backup is not default. Thanks for catching the misspelling >_<